src/Categories/Diagram/Coequalizer/Properties.agda

```diff
@@ -63,8 +63,325 @@ regular-is-coeq-kp {A} {B} f record { C = D ; h = h ; g = g ; coequalizer = coeq
+                  → (sq₂ : CommutativeSquare α g₁ g₂ β)               -- g₂ ∘ α ≈ β ∘ g₁
+                  → (coeq₁ : Coequalizer f₁ g₁)
+                  → (coeq₂ : Coequalizer f₂ g₂)
+                  → CommutativeSquare
+                      β (Coequalizer.arr coeq₁)
+                      (Coequalizer.arr coeq₂) (⇒MapBetweenCoeq α β sq₁ sq₂ coeq₁ coeq₂)
+  ⇒MapBetweenCoeqSq α β sq₁ sq₂ coeq₁ coeq₂ = universal₁
+    where
+      open Coequalizer coeq₁ renaming (isCoequalizer to isCoequalizer₁)
+      open IsCoequalizer isCoequalizer₁ renaming (universal to universal₁)
+
+open MapBetweenCoequalizers public
+
+CoeqOfIsomorphicDiagram : {A B : Obj} {f g : A ⇒ B} (coeq : Coequalizer f g )
+                        → {A' B' : Obj}
+                        → (a : A ≅ A') (b : B ≅ B')
+                        → Coequalizer (_≅_.from b ∘ f ∘ _≅_.to a) (_≅_.from b ∘ g ∘ _≅_.to a)
+CoeqOfIsomorphicDiagram {A} {B} {f} {g} coeq {A'} {B'} a b = record
+  { arr = arr ∘ _≅_.to b
+  ; isCoequalizer = record
+    { equality = begin
+        (arr ∘ _≅_.to b) ∘ _≅_.from b ∘ f ∘ _≅_.to a ≈⟨ assoc²γβ ⟩
+        (arr ∘ _≅_.to b ∘ _≅_.from b) ∘ f ∘ _≅_.to a ≈⟨ elimʳ (_≅_.isoˡ b) ⟩∘⟨refl ⟩
+        arr ∘ f ∘ _≅_.to a ≈⟨ extendʳ equality ⟩
+        arr ∘ g ∘ _≅_.to a ≈⟨ introʳ (_≅_.isoˡ b) ⟩∘⟨refl ⟩
+        (arr ∘ _≅_.to b ∘ _≅_.from b) ∘ g ∘ _≅_.to a ≈⟨ assoc²βγ ⟩
+        (arr ∘ _≅_.to b) ∘ _≅_.from b ∘ g ∘ _≅_.to a ∎
+    ; coequalize = coequalize'
+    ; universal =  λ {C} {h} {eq} → begin
+        h ≈⟨ switch-fromtoʳ b universal ⟩
+        (coequalize' eq ∘ arr) ∘ _≅_.to b ≈⟨ assoc ⟩
+        coequalize' eq ∘ (arr ∘ _≅_.to b) ∎
+    ; unique = λ {C} {h} {i} {eq} e → unique (⟺ (switch-tofromʳ b (begin
+        (i ∘ arr) ∘ _≅_.to b ≈⟨ assoc ⟩
+        i ∘ arr ∘ _≅_.to b ≈⟨ ⟺ e ⟩
+        h ∎)))
+    }
+  }
+  where
+    open Coequalizer coeq
+    open HomReasoning
+    open MR 𝒞
+    
+    f' g' : A' ⇒ B'
+    f' = _≅_.from b ∘ f ∘ _≅_.to a
+    g' = _≅_.from b ∘ g ∘ _≅_.to a
+
+    equalize'⇒equalize : {C : Obj} {h : B' ⇒ C}
+                         (eq : h ∘ f' ≈ h ∘ g')
+                       → (h ∘ _≅_.from b) ∘ f ≈ (h ∘ _≅_.from b) ∘ g
+    equalize'⇒equalize {C} {h} eq = cancel-toʳ a (begin
+      ((h ∘ _≅_.from b) ∘ f) ∘ _≅_.to a ≈⟨ assoc²αε ⟩
+      h ∘ f'                            ≈⟨ eq ⟩
+      h ∘ g'                            ≈⟨ assoc²εα ⟩
+      ((h ∘ _≅_.from b) ∘ g) ∘ _≅_.to a ∎)
+
+    coequalize' : {C : Obj} {h : B' ⇒ C}
+                  (eq : h ∘ f' ≈ h ∘ g')
+                → obj ⇒ C
+    coequalize' {C} {h} eq = coequalize (equalize'⇒equalize eq)
+
+
+-- coequalizer commutes with coequalizer
+module CoequalizerOfCoequalizer
+  {A B C D : Obj} {f₁ f₂ : A ⇒ B} {g₁ g₂ : A ⇒ C} {h₁ h₂ : B ⇒ D} {i₁ i₂ : C ⇒ D}
+  (coeqᶠ : Coequalizer f₁ f₂) (coeqᵍ : Coequalizer g₁ g₂)
+  (coeqʰ : Coequalizer h₁ h₂) (coeqⁱ : Coequalizer i₁ i₂)
+  (f⇒i₁ f⇒i₂ : Coequalizer.obj coeqᶠ ⇒ Coequalizer.obj coeqⁱ)
```

Comment:
maybe put `let open Coequalizer` in this telescope, so you can shorten the rest?